sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1774,6 +1775,34 @@ class Analyzer(
   }
 
   /**
+   * Substitute Hints.
+   */
+  object SubstituteHint extends Rule[LogicalPlan] {
+    def apply(plan: LogicalPlan): LogicalPlan = plan transform {
```

Comment:
Sorry for the wrong query (I didn't test it). My point is what will happen when we encounter multiple query hints? How are they processed?